scripts/modules/book_class/book_class.py

```diff
@@ -28,7 +29,9 @@ def login_and_book(self, class_id):
         'op': 'book-aulas'
       })
 
-      if resp.ok:
```

Comment:
If removing this line, the POST response on line 25 can be discarded. I know that we should check all answers for a 2xx type response at least.